src/components/VideoForm/VideoForm.jsx

```diff
@@ -118,6 +119,8 @@ export default function VideoForm() {
 				<StyledButton>Submit</StyledButton>
 			</StyledForm>
 			{showText ? <StyledSumbitText>Great! Your video was added</StyledSumbitText> : ''}
```

Comment:
Just a minor hint to avoid the condition with the empty string. Use the logical `&&` operator.
It works because `true && expression` always evaluates to expression, and `false && expression` always evaluates to false. If the condition is true, the element right after `&&` will appear. If it is false, it will ignored. 🧙‍♀️

```suggestion
			{showText && <StyledSumbitText>Great! Your video was added</StyledSumbitText> }
```

Reply:
@JessicaLoers  This is much better! :)